src/store/setup-key/actions.ts

```diff
@@ -28,13 +28,16 @@ const actions = {
       'DELETE_SETUP_KEY_FAILURE'
   )<RequestPayload<string>, DeleteResponse<string | null>, DeleteResponse<string | null>>(),
   setDeleteSetupKey: createAction('SET_DELETE_SETUP_KEY')<DeleteResponse<string | null>>(),
+  resetDeletedSetupKey: createAction('RESET_DELETE_SETUP_KEY')<null>(),
 
   revokeSetupKey: createAsyncAction(
       'REVOKE_SETUP_KEY_REQUEST',
       'REVOKE_SETUP_KEY_SUCCESS',
       'REVOKE_SETUP_KEY_FAILURE'
   )<RequestPayload<SetupKeyRevoke>, ChangeResponse<SetupKey | null>, ChangeResponse<SetupKey | null>>(),
-  setRevokeSetupKey: createAction('SET_REVOKE_SETUP_KEY')<ChangeResponse<SetupKey | null>>(),
+  setRevokeSetupKey: createAction('SET_REVOKED_SETUP_KEY')<ChangeResponse<SetupKey | null>>(),
+  resetRevokedSetupKey: createAction('RESET_REVOKED_SETUP_KEY')<null>(),
```

Comment:
Codacy has a fix for the issue: [Replace `'RESET_REVOKED_SETUP_KEY'` with `"RESET_REVOKED_SETUP_KEY"`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
  resetRevokedSetupKey: createAction("RESET_REVOKED_SETUP_KEY")<null>(),
```